### ISSUE TITLE
Missing Wolof letters

### ISSUE THREAD

**Issue by dwhitena** [keymanapp/keyboards#1624]
A contact in Senegal asked me about some potential missing letters in the Wolof keyboard. My intention is to add these myself if I can figure out how, but I thought I would at least create this issue (which I can hopefully close myself). Here is what the contact told me:

> Here are the missing letters on the keyboard:
> ë : ?
> é : French Keyboard ?
> õ : ?

**Comment by LornaSIL** [comment]
Are you talking about the `Wolof Basic` keyboard? We can't modify that keyboard since it's supposed to be an import from the Windows layout. 

The good (or bad) news is that it is not well documented and what you want is already available:

ë : [SHIFT K_LBRKT] + e = ë (that is `{e`)  (the `{` is what is called a deadkey and will work with a, e, i, o, u, y)
é : [K_2] = é
õ : [RALT K_2] + o = õ (the RALT 2 is what is called a deadkey and will work with a, n, o)

Leave this issue open and we will try to update the documentation.

**Comment by dwhitena** [comment]
Yes @LornaSIL, I think the `Wolof Basic` keyboard. Thanks so much for following up very quickly! I'll pass along this information to my contact. I would be happy to update the documentation once I figure out where/how!

**Comment by DavidLRowe** [comment]
@dwhitena The files for the Basic Wolof keyboard are located at:
https://github.com/keymanapp/keyboards/tree/master/release/basic/basic_kbdwol
The specific file to update is:
https://github.com/keymanapp/keyboards/blob/master/release/basic/basic_kbdwol/source/welcome/welcome.htm

Ideally what is needed is a list of all the characters that can be generated by this keyboard. Sometimes people use a chart with the deadkey down the left side and the second key across the top, with the result in the table cell. (Since deadkeys often work with only a few second keys, there are often many blank cells in the table.)

Another approach is to list each deadkey in a new paragraph, include an example, then specify which other keys the deadkey works with (as @LornaSIL did above). 

Thanks for pointing out this problem. If you'd like to contribute, let us know if you need more information on how to do that.
